src/modules/users/CMakeLists.txt

```
@@ -6,6 +6,16 @@
 find_package(${qtname} ${QT_VERSION} CONFIG REQUIRED Core DBus Network)
 find_package(Crypt REQUIRED)
 
+# Check for crypt_gensalt
+if(Crypt_FOUND)
+    list(APPEND CMAKE_REQUIRED_LIBRARIES crypt)
+    include(CheckSymbolExists)
+    check_symbol_exists(crypt_gensalt crypt.h HAS_CRYPT_GENSALT)
```

And undo afterwards:
```suggestion
    check_symbol_exists(crypt_gensalt crypt.h HAS_CRYPT_GENSALT)
    set(CMAKE_REQUIRED_LIBRARIES "${_old_CRL}")
```